pkg/cmd/bundle/list.go

```diff
@@ -113,8 +112,13 @@ Caching:
 				Err: cmd.OutOrStderr(),
 			}
 
+			p, err := f.ToPrinter()
```

Comment:
can we keep this also inside `PrintObjects` function, it is getting duplicated at multiple places

Reply:
Notice how `PrintObjects` was called for each item in the list, individually. This means a new instance of the printer is created for each call which causes the issue in the first place. I tried to setup some sort of caching where the printer would only be created once per output format. The unit tests revealed that to be a problematic pattern as it forces a global state between different commands.

Any suggestions?

Reply:
ok this makes sense, for now will go with this